WordPress/gutenberg
# Add command: View post

## What?

Fixes https://github.com/WordPress/gutenberg/issues/55806

## Why?

Currently view page/post command is not available.


## How?

Added command for viewing page/ post from command box.

## Testing Instructions

- Open the WordPress Editor
- Go to Posts or Pages in the WordPress admin.
- Open a new or existing post/page to enter the block editor.
- Confirm Command Labels for Published Content
- Publish the post or page if it's not already published.
- In the block editor, open the Command Palette (by pressing Cmd + K on Mac).
- Check for the following commands:
  - View page (if editing a page).
  - View post (if editing a post).
- Select the command and confirm it opens the published post or page in a new tab.
- If the post/page is not already in draft, click Save as Draft or Preview to set it to a non-published state.
- Open the Command Palette again and check for the following commands:
  - Preview page (if editing a page).
  - Preview post (if editing a post).
- Select the command and confirm it opens the preview of the draft post or page in a new tab.


## Screenshots or screencast

https://github.com/user-attachments/assets/3b3e2656-9866-4bf2-a389-09a4eb43123b
